libs/template/let/src/lib/tests/let.directive.strategy.spec.ts

```diff
@@ -73,7 +67,76 @@ describe('LetDirective when using strategy', () => {
   beforeAll(() => mockConsole());
   beforeEach(setupLetDirectiveTestComponentStrategy);
 
-  it('should work with different if a strategy other than the default', () => {
+  it('should work with default strategy', done => {
+    letDirectiveTestComponent.value$ = of(5);
+    fixtureLetDirectiveTestComponent.detectChanges();
+    setTimeout(() => {
+      expect(componentNativeElement.textContent).toBe('5');
+      done();
+    }, 1);
+  });
```

Comment:
What about using `fakeAsync` + `flush` to avoid `setTimeout` callback + `done` fn for all asynchronous tests? 

```ts
  it('should work with default strategy', fakeAsync(() => {
    letDirectiveTestComponent.value$ = of(5);
    fixtureLetDirectiveTestComponent.detectChanges();
    flush();
    expect(componentNativeElement.textContent).toBe('5');
  }));
```

Reply:
Hey Edouard. Would `await fixture.whenStable()` work in that case? I mean:
```ts
 it('should work with default strategy', async () => { 
  ...
  await ...
  expect ...
```

I'm only curious, haven't checked locally.

Reply:
Agree it also works and it's even better. 

```ts
  it('should work with default strategy', async () => {
    letDirectiveTestComponent.value$ = of(5);
    fixtureLetDirectiveTestComponent.detectChanges();
    await fixtureLetDirectiveTestComponent.whenStable();
    expect(componentNativeElement.textContent).toBe('5');
  });
```

Reply:
Sorry im a test newbie, thx for help :)

Reply:
@arturovt @edbzn implemented it. the error is still there. is there something wrong with my implementation?